src/components/InstrumentControls/Camera.vue

```diff
@@ -337,14 +353,23 @@ export default {
       'available_devices',
       'selected_camera_config',
       'camera_has_darkslide',
-      'camera_can_bin',
-      'camera_default_bin'
+      'camera_can_bin'
     ]),
 
     number_of_cameras () {
       return Object.keys(this.available_devices('camera', this.sitecode)).length
     },
 
+    smartstackIsActive: {
```

Comment:
The new toggles and options look good! I think we should add a few tooltips that explain what some of these new options are though.

Reply:
This might be a task @mfitzasp is better suited for - if not, I'm happy to take a crack at it!